x/tokenomics/keeper/update_relay_mining_difficulty.go

```diff
@@ -123,37 +121,37 @@ func ComputeNewDifficultyTargetHash(targetNumRelays, newRelaysEma uint64) []byte
 	// the actual on-chain relays, so we don't need to scale to anything above
 	// the default.
 	if targetNumRelays > newRelaysEma {
-		return defaultDifficultyTargetHash()
+		return prooftypes.DefaultRelayDifficultyTargetHash
 	}
 
-	log2 := func(x float64) float64 {
-		return math.Log(x) / math.Ln2
-	}
+	// Calculate the proportion of target relays to the new EMA
+	ratio := float64(targetNumRelays) / float64(newRelaysEma)
+
+	// Compute the new target hash by scaling the default target hash based on the ratio
+	newTargetHash := scaleDifficultyTargetHash(prooftypes.DefaultRelayDifficultyTargetHash, ratio)
 
-	// We are dealing with a bitwise binary distribution, and are trying to convert
-	// the proportion of an off-chain relay (i.e. relayEMA) to an
-	// on-chain relay (i.e. target) based on the probability of x leading zeros
-	// in the target hash.
-	//
-	// In other words, the probability of an off-chain relay moving into the tree
-	// should equal (approximately) the probability of having x leading zeroes
-	// in the target hash.
-	//
-	// The construction is as follows:
-	// (0.5)^num_leading_zeroes = (num_target_relay / num_total_relays)
-	// (0.5)^x = (T/R)
-	// 	x = -ln2(T/R)
-	numLeadingZeroBits := int(-log2(float64(targetNumRelays) / float64(newRelaysEma)))
-	numBytes := protocol.SmtSpec.PathHasherSize()
-	return LeadingZeroBitsToTargetDifficultyHash(numLeadingZeroBits, numBytes)
+	return newTargetHash
 }
 
-// defaultDifficultyTargetHash returns the default difficulty target hash with
-// the default number of leading zero bits.
-func defaultDifficultyTargetHash() []byte {
-	numBytes := protocol.SmtSpec.PathHasherSize()
-	numDefaultLeadingZeroBits := int(prooftypes.DefaultMinRelayDifficultyBits)
-	return LeadingZeroBitsToTargetDifficultyHash(numDefaultLeadingZeroBits, numBytes)
+// scaleDifficultyTargetHash scales the default target hash based on the given ratio
+func scaleDifficultyTargetHash(targetHash []byte, ratio float64) []byte {
```

Comment:
Can you add unit tests for this? It'll make it easier to understand what the function is doing.

I'm not sure what's going on at the momentt.